src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -231,7 +231,7 @@ private static void ValidateKindAndLevel(ResultKind kind, FailureLevel level)
 
             string messageText =
                    (path != null ? (path + location) : toolName) + ": " +
-                   issueType + (!string.IsNullOrEmpty(ruleId) ? " " : "") +
+                   issueType + " " +
```

Comment:
>issueType [](start = 19, length = 9)

Without this fix, in the absence of a rule id, the 'level' would be immediately followed by the message, e.g., 'errorHereis the message'.